unittest/lang_model_test.cc

```diff
@@ -111,4 +214,4 @@ TEST(LangModelTest, AddACharacter) {
 }
 
 }  // namespace
-}  // namespace tesseract
+}  // namespace tesseract
```

Comment:
Here you accidentally removed the terminating linefeed.

Reply:
Fixed via 5e90a61eb4e27d00d70a8385369bd969dd786df4
Thanks!